### PR TITLE
Fix test warning

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
# Description:

Fixes one test warning

```
/path/to/test/rubygems/test_gem_commands_build_command.rb:392: warning: ambiguous first argument; put parentheses or a space even after `/' operator
```

And turns warnings on for our tests.

# Tasks:

- [x] Describe the problem / feature
- [x] Write tests
- [x] Write code to solve the problem
- [ ] Get code review from coworkers / friends

I will abide by the [code of conduct](https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: test/rubygems/test_gem_commands_build_command.rb [hunk 1]
```diff
@@ -389,7 +389,7 @@ def test_build_auto_resign_cert
     output = @ui.output.split "\n"
     assert_equal "INFO:  Your certificate has expired, trying to re-sign it...", output.shift
     assert_equal "INFO:  Your cert: #{tmp_expired_cert_file } has been auto re-signed with the key: #{tmp_private_key_file}", output.shift
-    assert_match /INFO:  Your expired cert will be located at: .+\Wgem-public_cert\.pem\.expired\.[0-9]+/, output.shift
+    assert_match(/INFO:  Your expired cert will be located at: .+\Wgem-public_cert\.pem\.expired\.[0-9]+/, output.shift)
```
Comment (bronzdoc) [inline test/rubygems/test_gem_commands_build_command.rb]: Why this change in particular?

Comment (deivid-rodriguez) [reply test/rubygems/test_gem_commands_build_command.rb]: Because it's what the warning is telling me to do... Otherwise the code is ambigous because `/` could also mean integer division... I guess? :thinking:. Not really sure where the ambiguity lies, I just know you fix the warning by adding parentheses :laughing:

Comment (bronzdoc) [reply test/rubygems/test_gem_commands_build_command.rb]: Ah i see, sorry i didn't read well the description of the issue 😂

**Comment by bronzdoc** [comment]
@bundlerbot r+
